src/index.html

```diff
@@ -15,8 +15,39 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="container">
+        <a
+          href="https://th.bing.com/th/id/OIP.axjC06-qxFXElNPCbM1MPAAAAA?rs=1&pid=ImgDetMain"
+          target="_blank"
+        >
```

Comment:
Don't use spaces in the `href` property of the `a` tag. Anchor links should start with the `#` symbol unless it is a link to a different webpage, which seems to be the case here. However, the `href` value appears to be an image source rather than a webpage URL, which is not typical for navigation links.